packages/driver/src/rawConn.ts

```diff
@@ -335,7 +335,7 @@ export class RawConnection extends BaseRawConnection {
             );
             break;
         }
-        err.source = e;
+        err.cause = e;
```

Comment:
I think this duplicates the logic above?

Reply:
lol, I added that for debugging and it made it into `master` and I totally forgot about it 🤦 will fix.